x-pack/legacy/plugins/apm/public/hooks/useFetcher.tsx

```diff
@@ -57,7 +61,14 @@ export function useFetcher<Response>(
           });
         }
       } catch (e) {
+        const err = e as KFetchError;
         if (!didCancel) {
+          toastNotifications.addWarning({
+            title: i18n.translate('xpack.apm.fetcher.error.title', {
+              defaultMessage: `Error while fetching resource`
+            }),
+            text: `${idx(err.res, r => r.status)}: ${idx(err.res, r => r.url)}`
```

Comment:
I feel like this could be a bit more user friendly. Maybe a little markup? Do we need the status code? Is there any property from the error (response) we can display instead? (e.g., often errors have a `message` property that we can surface).

Reply:
There is also the `error.statusText` which we can use instead of `status`. So instead of `404` it would show `Not found`. It is not always as distinguishable as a status code (I'm thinking if the user takes a screenshot and opens a support ticket) but I'm okay with changing it.

Reply:
Full list of status codes and status texts: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Looks like the texts are unique so might be as good as a status code for us.

Reply:
Added some markup and the statusText. Looks a whole bit nicer now :)

<img width="350" alt="Screen Shot 2019-07-31 at 11 59 39" src="https://user-images.githubusercontent.com/209966/62203092-d0b6a400-b38a-11e9-83a7-8a36bdc3f905.png">